src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -592,7 +592,7 @@ export default {
 				}
 			}
 
-			if (this.text !== '') {
+			if (this.hasText) {
 				// FIXME: remove after issue is resolved: https://github.com/nextcloud/nextcloud-vue/issues/3264
 				const temp = document.createElement('textarea')
 				temp.innerHTML = this.text
```

Comment:
```suggestion
				temp.innerHTML = this.text.trim()
```
We trim messages at server, probably, so temporary message could be aligned to it as well

Reply:
Fixed